src/site/stages/build/index.js

```diff
@@ -39,6 +46,40 @@ const rewriteVaDomains = require('./plugins/rewrite-va-domains');
 const updateExternalLinks = require('./plugins/update-external-links');
 const updateRobots = require('./plugins/update-robots');
 
+/**
+ * Immediately copies the Webpack build output to a directory outside of
+ * Metalsmith's build destination, then returns a function for use as a
+ * Metalsmith plugin. This plugin copies the files back to their expected
+ * location.
+ *
+ * This is needed because script/build.sh runs Webpack before the content build,
+ * and Metalsmith's build method removes everything in the destination, which
+ * wipes out the output of the Webpack build.
+ *
+ * This can be removed when we move the content build to a new repository and
+ * this script no longer interacts with the Webpack output at all.
+ */
+function preserveWebpackOutput(metalsmithDestination) {
```

Comment:
If the comment for this function doesn't make sense, please let me know how I can improve it.